poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
+            </div>
+            {{#if any_img}}
+                <div class="poll-image result-image">
+                    <label for="answer-{{key}}" class="poll-image-label">
+                        {{#if img}}
+                          <img src="{{img}}" />
```

Comment:
All images *MUST* have an ```alt``` attribute (these images do not).  While it is acceptable in some contexts to have an null alt (```alt=""```) this would not be one of them seeing as though some description of the image would be required to differentiate one choice from the other.
http://www.w3.org/html/wg/drafts/html/CR/embedded-content-0.html#alt

There must be a mechanism for the poll's author to provide a description for each image.  I recommend adding text that helps the author understand the meaning of useful alternatives (i.e. "Describe this image in a way that would allow someone to answer the poll if the image did not load")